src/main/java/edu/njit/jerse/ashe/Ashe.java

```diff
@@ -163,11 +162,12 @@ public static void run(String root, String targetFile, String targetMethod, Stri
             }
         } finally {
             LOGGER.info("Cleaning up temporary directory: " + speciminTempDir);
+            /**
```

Comment:
If this should be removed, it's better to just delete it rather than commenting it out.

Reply:
will revert - this was unintended(commented out so the specimin output could be viewed for testing)